yarn.lock

```diff
@@ -6832,7 +6821,18 @@ ember-cli-htmlbars-inline-precompile@^2.1.0:
     heimdalljs-logger "^0.1.9"
     silent-error "^1.1.0"
 
-ember-cli-htmlbars@^4.0.0, ember-cli-htmlbars@^4.0.8, ember-cli-htmlbars@^4.0.9, ember-cli-htmlbars@^4.2.0, ember-cli-htmlbars@^4.2.2, ember-cli-htmlbars@^4.2.3, ember-cli-htmlbars@^4.3.1:
+ember-cli-htmlbars@^3.0.0, ember-cli-htmlbars@^3.0.1, ember-cli-htmlbars@^3.1.0:
+  name ember-cli-htmlbars-3
```

Comment:
This `name` alias is what is causing the tests to continue to pass. Someone in the build has `ember-cli-htmlbars@3` still, and when `yarn` was ran after removing the alias it did **not** remove the `name` field here.

Removing this `name` field makes the tests properly fail.